packages/config/config/devices/0x0148/cometz_700.json

```diff
@@ -80,6 +80,17 @@
 				}
 			]
 		},
+		{
+			// Some devices report this param, some don't, may be related to firmware version.
+			// But even when not reported by the device, it is supported.
+			"#": "6",
+			"label": "Valve Opening Percentage Report",
```

Comment:
Is this a threshold, meaning if this is set to 5%, the device reports every change over 5%?

Reply:
Yes